beautysh/beautysh.py

```diff
@@ -5,6 +5,22 @@
 import re
 import sys
 
+# correct function style detection is obtained only if following regex are tested in sequence.
+# styles are listed as follows:
+# 0) function keyword, open/closed parentheses
```

Comment:
I think these are more instructive if you show an example, like
`function foo()`
`function foo`
`foo()`

Reply:
you're right, added!